mappings/net/minecraft/world/biome/source/util/VanillaBiomeParameters.mapping

```diff
@@ -31,22 +31,38 @@ CLASS net/minecraft/class_6554 net/minecraft/world/biome/source/util/VanillaBiom
 		ARG 0 noisePoint
 		ARG 2 noiseRanges
 	METHOD method_38181 getShoreBiome (II)Lnet/minecraft/class_5321;
+		COMMENT {@return an appropriate shore biome for the given temperature and humidity}.
 		ARG 1 temperature
 		ARG 2 humidity
 	METHOD method_38182 getRegularBiome (IILnet/minecraft/class_6544$class_6546;)Lnet/minecraft/class_5321;
+		COMMENT {@return a regular biome.} This can be a {@linkplain #commonBiomes common} or {@linkplain #uncommonBiomes uncommon} biome, depending on
+		COMMENT temperature and humidity.
+		COMMENT
+		COMMENT Note that for negative weirdness values, only common biomes can get picked by this
```

Comment:
Is this a new paragraph?

Reply:
I guess. New paragraphs need the `<p>` tag, right? Should I add that here?

Reply:
Yeah.